src/resources/resources.css

```diff
@@ -83,7 +83,46 @@
 
 .pagination-container {
 	margin: auto;
+}
+
+/* List Group Buttons To List Items */
+.resource-list {
 	display: flex;
+	flex-direction: column;
+	padding-left: 0rem;
+	margin-bottom: 0.5rem;
+}
+
+.resource-list-title {
+	position: relative;
+	text-align: left;
+	display: block;
+	padding: 0.75rem;
+	margin-bottom: -0.1rem;
+	color: white;
+	background-color: black;
```

Comment:
I think we should remove background-color for the title. It looks uncomfortable. I suggest underlining and/or addind bullet the title.

Reply:
How's this?
<img width="684" alt="Screen Shot 2022-08-15 at 7 57 28 PM" src="https://user-images.githubusercontent.com/36748732/184788801-5b9ffcb4-29ff-433e-b1d0-f7861497f855.png">

Reply:
It looks pretty good. I suggest adding more space between each tutorial container. After expanding, I think you should remove bottom border of grey box title.

Reply:
Yeah I was already thinking it needed more space. Removing the bottom border is a good change!